src/qt/askpassphrasedialog.h

```diff
@@ -26,6 +26,7 @@ class AskPassphraseDialog : public QDialog
         Encrypt,    /**< Ask passphrase twice and encrypt */
         Unlock,     /**< Ask passphrase and unlock */
         ChangePass, /**< Ask old passphrase + new passphrase twice */
+        UnlockMigration, // Ask passphrase for unlocking during migration
```

Comment:
Mind following the Doxygen comment formatting above?

Reply:
Done